features/search/works_collected.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Search collected works
+  As a creator of collected works
+  I want to filter for works across all those collections
+
+  Scenario: Works that are collected show up in the creator's
+  collected works page
+    Given I am logged in as "author"
+      And I create the collection "Author Collection"
+      And I create the collection "Other Collection"
+      And I post the work "Old Title" to the collection "Author Collection"
+      And I post the work "Revised Title" to the collection "Author Collection"
+      And I post the work "New Title" to the collection "Other Collection"
+      And I post a chapter for the work "Revised Title"
```

Comment:
Please add a `And I wait 1 second` before both of these two steps. Otherwise, if the test runs especially quickly, the works can end up with the same timestamp which messes with the sorting.